crates/solidity/outputs/cargo/crate/src/backend/built_ins.rs

```diff
@@ -165,12 +166,21 @@ pub enum BuiltIn {
     YulStop,
     YulSub,
     YulSuicide,
+    YulSwap(u8),
     YulTimestamp,
     YulTload,
     YulTstore,
     YulXor,
 }
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum LegacyCall {
+    Call,
+    Delegatecall,
+    Staticcall,
+    User(TypeId),
+}
+
 const VERSION_0_4_12: Version = Version::new(0, 4, 12);
```

Comment:
nit:
this module, most passes, the parser, and even tests is creating these `const` versions for comparison. Should we generate them in a single module instead of maintaining individual lists?